src/pages/settings/Wallet/PaymentMethodList.js

```diff
@@ -69,6 +72,9 @@ const propTypes = {
     /** React ref being forwarded to the PaymentMethodList Button */
     buttonRef: PropTypes.oneOfType([PropTypes.func, PropTypes.object]),
 
+    /** To enable/disable scrolling */
+    scrollEnabled: PropTypes.bool,
```

Comment:
Probably more common to see the `shouldEnableScroll` convention like we have for the other optional behavior